[2022-12-18T00:38:48.314+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: postgres_with_airflow.create_table_task manual__2022-12-17T06:22:10.720013+00:00 [queued]>
[2022-12-18T00:38:48.428+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: postgres_with_airflow.create_table_task manual__2022-12-17T06:22:10.720013+00:00 [queued]>
[2022-12-18T00:38:48.429+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-18T00:38:48.430+0000] {taskinstance.py:1284} INFO - Starting attempt 3 of 3
[2022-12-18T00:38:48.433+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-18T00:38:48.633+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table_task> on 2022-12-17 06:22:10.720013+00:00
[2022-12-18T00:38:48.719+0000] {standard_task_runner.py:55} INFO - Started process 787 to run task
[2022-12-18T00:38:48.865+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'postgres_with_***', 'create_table_task', 'manual__2022-12-17T06:22:10.720013+00:00', '--job-id', '58', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpgr6fbc1j']
[2022-12-18T00:38:48.873+0000] {standard_task_runner.py:83} INFO - Job 58: Subtask create_table_task
[2022-12-18T00:38:49.977+0000] {task_command.py:389} INFO - Running <TaskInstance: postgres_with_airflow.create_table_task manual__2022-12-17T06:22:10.720013+00:00 [running]> on host f376bb2680d1
[2022-12-18T00:38:52.107+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_with_***
AIRFLOW_CTX_TASK_ID=create_table_task
AIRFLOW_CTX_EXECUTION_DATE=2022-12-17T06:22:10.720013+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-17T06:22:10.720013+00:00
[2022-12-18T00:38:52.249+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE traffic_data(id INT NOT NULL, track_id INT NOT NULL, type VARCHAR(250) NOT NULL, traveled_d VARCHAR(250) NOT NULL, avg_speed VARCHAR(250) NOT NULL, lat VARCHAR(250) NOT NULL, lon VARCHAR(250) NOT NULL, speed VARCHAR(250) NOT NULL, lon_acc VARCHAR(250) NOT NULL, lat_acc VARCHAR(250) NOT NULL, time VARCHAR(250) NOT NULL);
        
[2022-12-18T00:38:52.621+0000] {base.py:73} INFO - Using connection ID '***_dbt_setup' for task execution.
[2022-12-18T00:39:11.838+0000] {base.py:73} INFO - Using connection ID '***_dbt_setup' for task execution.
[2022-12-18T00:39:11.884+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE traffic_data(id INT NOT NULL, track_id INT NOT NULL, type VARCHAR(250) NOT NULL, traveled_d VARCHAR(250) NOT NULL, avg_speed VARCHAR(250) NOT NULL, lat VARCHAR(250) NOT NULL, lon VARCHAR(250) NOT NULL, speed VARCHAR(250) NOT NULL, lon_acc VARCHAR(250) NOT NULL, lat_acc VARCHAR(250) NOT NULL, time VARCHAR(250) NOT NULL);
        , parameters: None
[2022-12-18T00:39:12.004+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=postgres_with_***, task_id=create_table_task, execution_date=20221217T062210, start_date=20221218T003848, end_date=20221218T003912
[2022-12-18T00:39:12.169+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-18T00:39:12.323+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
